dvc/command/experiments.py

```diff
@@ -384,21 +384,9 @@ def baseline_styler(typ):
     return {"style": "bold"} if typ == "baseline" else {}
 
 
-def show_experiments(
-    all_experiments, pager=True, no_timestamp=False, **kwargs
-):
-    include_metrics = _parse_filter_list(kwargs.pop("include_metrics", []))
-    exclude_metrics = _parse_filter_list(kwargs.pop("exclude_metrics", []))
-    include_params = _parse_filter_list(kwargs.pop("include_params", []))
-    exclude_params = _parse_filter_list(kwargs.pop("exclude_params", []))
-
-    metric_names, param_names = _collect_names(
-        all_experiments,
-        include_metrics=include_metrics,
-        exclude_metrics=exclude_metrics,
-        include_params=include_params,
-        exclude_params=exclude_params,
-    )
+def _experiments_td(
```

Comment:
I'd prefer not to have too many layers here. Initially, I divided it into `show_experiments` (that knows about CLI flags and knows how to render/style) and `experiments_table` that simply parses and does not care about CLI.